Apps/Playground/CMakeLists.txt

```diff
@@ -175,6 +175,11 @@ if(APPLE)
             XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES
             RESOURCE "${RESOURCE_FILES}"
             FOLDER "Playground")
+        # The NSDocumentRevisionsDebugMode argument is typically added automatically by Xcode when running a macOS application in debug mode.
```

Comment:
nit: newline before this would be good for readability

Reply:
Maybe also add comment on why we are disabling?

Reply:
Fixed